ios/KochavaPlugin.mm

```diff
@@ -26,7 +26,9 @@ - (void) initializeWithManifest:(NSDictionary *)manifest
                               KOCHAVA_APP_ID, @"kochavaAppId",
                               nil];
 
-    self.kochavaTracker = [[KochavaTracker alloc] initKochavaWithParams:initDict];
+    self.kochavaTracker = [[KochavaTracker alloc]
+      initWithParametersDictionary:@{kKVAParamAppGUIDStringKey:
+      KOCHAVA_APP_ID} delegate:nil];
```

Comment:
what delegates are available @anil-kumar07  ?

Reply:
@rampr 
I followed the standard method of initialising from the doc.

Ex:

// start your own instance of class 
`KochavaTrackerKochavaTracker *tracker = [[KochavaTracker alloc] initWithParametersDictionary:@{kKVAParamAppGUIDStringKey: @"_YOUR_APP_GUID_"} delegate:nil];`

Reply:
@rampr delegate is nil here